### PR TITLE
feat(#1136); add missing toast messages

### PR DESCRIPTION
**Pull request by danielsteinkogler**
## Basics

- [x] The PR is rebased with current master
- [x] I added a line to [changelog.md](/doc/changelog.md)
- [x] Details of what I changed are in the commit messages
- [x] References to issues, e.g. `close #X`, are in the commit messages and changelog
- [x] The buildserver is happy

## Checklist

- [ ] I fully described what my PR does in the documentation
- [ ] I fixed all affected documentation
- [ ] I fixed the introduction tour
- [ ] I wrote migrations in a way that they are compatible with already present data
- [ ] I fixed all affected decisions
- [ ] I added automated tests or a [manual test protocol](../doc/tests/manual/protocol.md)
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I translated all strings visible to the user
- [ ] I mentioned [every code or binary](https://github.com/ElektraInitiative/PermaplanT/blob/master/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)
- [ ] I created left-over issues for things that are still to be done
- [ ] Code is conforming to [our Architecture](/doc/architecture)
- [ ] Code is conforming to [our Guidelines](/doc/guidelines)
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)
- [ ] Exceptions to any guidelines are documented

## First Time Checklist

- [ ] I have installed and I am using [pre-commit hooks](../doc/contrib/README.md#Hooks)
- [ ] I am using [Tailwind CSS Linting](https://tailwindcss.com/blog/introducing-linting-for-tailwindcss-intellisense)

## Review

- [ ] I've tested the code
- [x] I've read through the whole code
- [ ] I've read through the whole documentation
- [x] I've checked conformity to guidelines
- [x] I've checked conformity to requirements
- [ ] I've checked that the requirements are tested

### CODE REVIEW

**Comment by danielsteinkogler** [comment]
@Bushuo 
part of the issue is also to remove "autoClose". DId you mean that "autoClose: false" should be removed so that toasts are closed automatically or do you mean that they shouldn't close automatically

**Comment by danielsteinkogler** [comment]
jenkins build please

**Comment by Bushuo** [comment]
@danielsteinkogler 
I think I meant to evaluate all query APIs again to see if they really should be closing automatically or not.
I can not really remember anymore what exactly I meant with this comment.
Sorry 😓

File: frontend/src/config/i18n/de/guidedTour.json [hunk 1]
```diff
@@ -8,6 +8,8 @@
     "confirmation_pause": "Pausieren",
     "confirmation_quit": "Beenden",
     "fetch_status_error": "Sorry, ich habe Schwierigkeiten den Status deiner Einführung in den Map Editor zu empfangen. Bitte lade die Seite neu, damit ich es noch einmal versuchen kann.",
+    "update_tour_status_error": "Sorry, ich habe Schwierigkeiten den Status deiner Einführung in den Map Editor zu aktualisieren.",
```
Comment (markus2330) [inline frontend/src/config/i18n/de/guidedTour.json:11]: Could you actually trigger this error? Is reloading of the page maybe useful?

```suggestion
    "update_tour_status_error": "Entschuldigung, ich habe Schwierigkeiten den Status deiner Einführung zu aktualisieren.",
```

File: frontend/src/config/i18n/de/guidedTour.json [hunk 2]
```diff
@@ -8,6 +8,8 @@
     "confirmation_pause": "Pausieren",
     "confirmation_quit": "Beenden",
     "fetch_status_error": "Sorry, ich habe Schwierigkeiten den Status deiner Einführung in den Map Editor zu empfangen. Bitte lade die Seite neu, damit ich es noch einmal versuchen kann.",
```
Comment (markus2330) [inline frontend/src/config/i18n/de/guidedTour.json:10]:
```suggestion
    "fetch_status_error": "Entschuldigung, ich habe Schwierigkeiten den Status deiner Einführung in den Map Editor zu empfangen. Bitte lade die Seite neu, damit ich es noch einmal versuchen kann.",
```

File: frontend/src/config/i18n/de/guidedTour.json [hunk 3]
```diff
@@ -8,6 +8,8 @@
...
+    "update_tour_status_error": "Sorry, ich habe Schwierigkeiten den Status deiner Einführung in den Map Editor zu aktualisieren.",
+    "complete_tour_error": "Sorry, ich habe Schwierigkeiten die Einführungstour abzuschließen.",
```
Comment (markus2330) [inline frontend/src/config/i18n/de/guidedTour.json:12]:
```suggestion
    "complete_tour_error": "Entschuldigung, ich habe Schwierigkeiten die Einführungstour abzuschließen.",
```

File: frontend/src/config/i18n/de/maps.json [hunk 4]
```diff
@@ -1,4 +1,5 @@
 {
+    "error_fetch_map_data": "Sorry, ich habe Schwierigkeiten die Karten-Daten von meinem Server zu erhalten. Bitte versuche es später erneut.",
```
Comment (markus2330) [inline frontend/src/config/i18n/de/maps.json:2]:
```suggestion
    "error_fetch_map_data": "Entschuldigung, ich habe Schwierigkeiten die Karten von meinem Server zu erhalten. Bitte versuche es später erneut.",
```

File: frontend/src/config/i18n/de/plantRelations.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
+  "error_fetching_relations": "Sorry, ich habe Schwierigkeiten die Beziehungen zwischen den Plfanzen zu laden. Bitte versuche es später noch einmal."
```
Comment (markus2330) [inline frontend/src/config/i18n/de/plantRelations.json:2]:
```suggestion
  "error_fetching_relations": "Entschuldigung, ich habe Schwierigkeiten die Beziehungen zwischen den Pflanzen zu laden. Bitte versuche es später noch einmal."
```

File: frontend/src/config/i18n/en/guidedTour.json [hunk 6]
```diff
@@ -8,6 +8,8 @@
     "confirmation_pause": "Pause",
     "confirmation_quit": "End",
     "fetch_status_error": "Sorry, I'm experiencing trouble fetching the status of your introduction to the Map Editor. Please reload the page so that I can try again.",
+    "update_tour_status_error": "Sorry, I'm experiencing trouble updating the status of your introduction to the Map Editor.",
```
Comment (markus2330) [inline frontend/src/config/i18n/en/guidedTour.json:11]:
```suggestion
    "update_tour_status_error": "Sorry, I'm experiencing trouble updating the status of your introduction tour.",
```

File: frontend/src/config/i18n/en/guidedTour.json [hunk 7]
```diff
@@ -8,6 +8,8 @@
...
+    "complete_tour_error": "Sorry, I'm experiencing trouble completing the introduction to the Map Editor.",
```
Comment (markus2330) [inline frontend/src/config/i18n/en/guidedTour.json:12]:
```suggestion
    "complete_tour_error": "Sorry, I'm experiencing trouble completing the introduction tour.",
```

File: frontend/src/config/i18n/en/maps.json [hunk 8]
```diff
@@ -1,5 +1,6 @@
 {
-    "overview": {
+  "error_fetch_map_data": "Sorry, I'm experiencing trouble fetching map data from my server. Please try again later.",
```
Comment (markus2330) [inline frontend/src/config/i18n/en/maps.json:2]:
```suggestion
  "error_fetch_map_data": "Sorry, I'm experiencing trouble fetching maps from my server. Please try again later.",
```

File: frontend/src/config/i18n/en/plantRelations.json [hunk 9]
```diff
@@ -0,0 +1,3 @@
+{
+  "error_fetching_relations": "Sorry, I'm experiencing trouble fetching plant relations from the server. Please try again later."
```
Comment (markus2330) [inline frontend/src/config/i18n/en/plantRelations.json:2]:
```suggestion
  "error_fetching_relations": "Sorry, I'm experiencing trouble fetching plant relations from my server. Please try again later."
```

**Review by markus2330 — CHANGES_REQUESTED** [review]
Great job, only a few minor text changes. If it is actual "map data" and not "maps", please keep your text.

**Comment by markus2330** [comment]
> part of the issue is also to remove "autoClose".

Yes, the default shouldn't be changed so that all toasts close automatically.

**Comment by danielsteinkogler** [comment]
jenkins build plesae

**Comment by danielsteinkogler** [comment]
jenkins build please

**Comment by danielsteinkogler** [comment]
jenkins build please

**Comment by markus2330** [comment]
great job, thank you :heart:

**Pull request merged**